new-hunt-setup.md

```diff
@@ -1,14 +1,14 @@
-## Setting up smallboard for a new hunt
+## Setting up cardboard for a new hunt
 
-Currently, smallboard is not multi-tenant (see [#191](https://github.com/cardinalitypuzzles/smallboard/issues/191)), so each team for the same hunt needs their own smallboard instance.
+Currently, cardboard is not multi-tenant (see [#191](https://github.com/cardinalitypuzzles/cardboard/issues/191)), so each team for the same hunt needs their own cardboard instance.
```

Comment:
Nit: should cardboard be capitalized as Cardboard?
Same elsewhere in this file.